shared/src/test/resources/vocabularies2/instances/native-link-with-extra-properties/instance.yaml

```diff
@@ -0,0 +1,7 @@
+#%Test 1.0
+description: this is node A
+link:
+  myVar: Jorge
+  a: 2
+  b: 2
+  # This link defines value for some of the properties not present in the template id
```

Comment:
@antoniogarrote Should we validate this case where the link defines some properties from the mapping that are not used for the template id?